.golangci.yml

```diff
@@ -715,3 +716,6 @@ linters-settings:
     pkgconfigusage:
       type: "module"
       description: "Check usage of pkgconfig in components folder"
+    forbiddenusage:
```

Comment:
Not: I think the name could be change to `deprecatedusage` 😄

Reply:
Since we have `depguard` we could name it `funcguard` 😁

Reply:
I renamed to `funcguard`